profile-googlefonts/src/checks/googlefonts/name/familyname_first_char.rs

```diff
@@ -0,0 +1,34 @@
+use fontspector_checkapi::{prelude::*, testfont, FileTypeConvert};
+use read_fonts::TableProvider;
+use skrifa::string::StringId;
+
+#[check(
+    id = "googlefonts/name/familyname_first_char",
+    rationale = "
+        
+        Font family names which start with a numeral are often not discoverable
+        in Windows applications.
+    
+    ",
+    proposal = "https://github.com/fonttools/fontbakery/issues/4829",
+    title = "Make sure family name does not begin with a digit."
+)]
+fn familyname_first_char(t: &Testable, _context: &Context) -> CheckFnResult {
+    let f = testfont!(t);
+    let mut problems = vec![];
+    let name = f.font().name()?;
+    for record in name.name_record().iter() {
```

Comment:
Same comment here, either use `get_name_entry_strings` or `best_familyname`.

Reply:
It will be `get_name_entry_strings` because we want to validate all family name entries.